health/service/src/main/java/com/cerner/beadledom/health/internal/HealthChecker.java

```diff
@@ -31,6 +31,7 @@
  */
 public class HealthChecker {
```

Comment:
Does a test need to be updated to test that your changes work?

Reply:
How can we test loggers?

Reply:
Existing tests cover the other refactorings I did, but I don't know how to test a log message.

Reply:
The [slf4j-test](http://projects.lidalia.org.uk/slf4j-test) project looks interesting for this, but we will lose our regular log output during tests if we use it.

Reply:
don't worry about it.